Documentation/articles/getting_started/0_getting_started.md

```diff
@@ -23,3 +21,23 @@ By the end of this tutorial set, you will have a working project to build off of
 - [Understanding the Code](3_understanding_the_code.md)
 - [Adding Content](4_adding_content.md)
 - [Adding Basic Code](5_adding_basic_code.md)
+
```

Comment:
@SimonDarksideJ @harry-cpp @Jjagg 

I think this just confuses things.  

We have a very simple clear document talking about getting started with MG.

I don't see the benefit to distract the user by listing all this other stuff.  A new user... one that needs this guide... doesn't care about any of this stuff.

Reply:
Except the "Downloads" page doesn't really server any purpose, so the suggestion was to merge in any additional information in to inform the user.

Thy can choose to follow or not, but at least they are informed and all the information is in one location instead of posted in several different locations.

Reply:
I agree that things seem quite clear as they are. 
I would however see value in a separate advanced topic page to detail how to use MonoGame from source and development nugets (which could be linked from the download page; I find the info on the download page to be a bit lacking on those topics, but I would also agree that this page needs to be small and straightforward, hence the suggestion to separate these in advanced topics).

Reply:
In which case then @mrhelmut would it be better to update the "Downloads" page with better links to the "Getting Started" page, since installing the Setup installer is only needed on Windows.

Maybe a grid for each Operating system, something like this

![image](https://user-images.githubusercontent.com/1793042/123959017-9a826480-d9a5-11eb-8b6f-3229e90316ea.png)